tests/integration/cloud/rest_contracts/test_data_context_configuration.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+import os
+import pathlib
+from typing import Callable, Final
+
+import pytest
+from pact import Like
+
+from tests.integration.cloud.rest_contracts.conftest import ContractInteraction
+
+ORGANIZATION_ID: Final[str] = os.environ.get("GX_CLOUD_ORGANIZATION_ID")
```

Comment:
Same note with error handling here - how do we inform devs if they're missing this env var?

Reply:
Additionally, I thought we had a custom `ruff` rule that disallows `os.environ` usage outside of particular files - @tyler-hoffman might know more?

Reply:
✅

Reply:
I did that `os.environ` change here: https://github.com/great-expectations/great_expectations/pull/8588

It doesn't seem to be applied to `tests` directory. @tyler-hoffman @Kilo59 we are only concerned with accessing `os.environ` in prod code right now, or should we be trying to banish it from `tests` as well?

Reply:
I don't think the use of `os.environ` in tests is necessarily a problem.